packaging/build.gradle

```diff
@@ -142,6 +141,7 @@ task prepareRelease(type: Copy) {
     from configurations.platformsRelease
     dependsOn createChecksums, signAllSrcZip, signAllDistZip
     from file("$buildDir/distributions")
+    from file("$buildDir/checksums")
```

Comment:
shouldn't it be `platformsRelease` artifact instead?

Reply:
We take the files from distributions and checksums folders directly since they are not declare outputs correctly.